tests/Unit/SDK/Common/Services/LoaderTest.php

```diff
@@ -88,15 +95,13 @@ public static function logRecordExporterProvider(): array
     #[DataProvider('textMapPropagator')]
     public function test_default_text_map_propagator(string $name): void
     {
-        $propagator = Registry::textMapPropagator($name);
+        $propagator = Loader::textMapPropagator($name);
         $this->assertInstanceOf(TextMapPropagatorInterface::class, $propagator);
     }
 
     public static function textMapPropagator(): array
     {
         return [
-            ['tracecontext'],
-            ['baggage'],
```

Comment:
Why were these removed?

Reply:
Lost in translation, added back and registered through SPI